docs/A-Introduction/information.md

```diff
@@ -78,33 +87,44 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
 
 The memory model for organizing information stored in RAM is linear.  Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
 
+<span className="mdImg" align="center" alt="A memory model image">
+
 ![RAM memory model: linearly mapped \(continuous sequence\)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
 
+</span>
+
 ### Addresses
 
 Each byte of RAM has a unique address.  Addressing starts at zero, is sequential, and ends at the address equal to the size of RAM less 1 unit. 
 
 For example, 4 Gigabytes of RAM
 
-* consists of 32 (= 4 * 8) Gigabits
+* consists of 32 (= 4 \* 8) Gigabits
 * starts at a low address of 0x00000000
 * ends at a high address of 0xFFFFFFFF
```

Comment:
I think address values in these lines should be wrapped in backticks.

Reply:
I have modified it, thx.

Reply:
You haven't.  Please make the correction.

Reply:
I have modified, thx